docs/ru/installation.md

```diff
@@ -36,7 +36,7 @@ return [
                 // Пример настроек подключения к SQLite:
                 'sqlite' => [
                     'driver' => \Spiral\Database\Driver\SQLite\SQLiteDriver::class,
-                    // Синтаксис подключения описан в https://www.php.net/manual/en/pdo.construct.php, смотрите DSN
+                    // Синтаксис подключения описан в https://www.php.net/manual/ru/pdo.construct.php, смотрите DSN
```

Comment:
```suggestion
                    // Синтаксис подключения описан в https://www.php.net/manual/pdo.construct.php, смотрите DSN
```
It is not at all necessary to specify the locale in the link to the documentation

Reply:
Done.